android/playground/src/main/java/com/kiwi/rnandroidplayground/HotelsData.kt

```diff
@@ -8,8 +8,8 @@ object HotelsData {
   fun getInitialProperties(): RNHotelsInitialProperties {
     return RNHotelsInitialProperties(
         "en",
-        "2018-12-01",
-        "2018-12-25",
+        "EUR",
+        "",
         "2018-12-20",
         "2018-12-25",
```

Comment:
Maybe change hardcoded dates for `Today` and `Tomorrow` dates. After 25.12 there will be `no results`.